modules/login/src/main/resources/META-INF/resources/login/js/login.js

```diff
@@ -0,0 +1,64 @@
+var loginpage = function($, w, moment) {
+
+    var pageData = {};
+
+    function setPageDate(data) {
+        pageData = data;
+    }
+    function switchWorkspace(selectobj) {
+        // document.getElementById("currentWorkspace").value = newWorkspace;
```

Comment:
I really didn't refactor the login javascript other than to move MIT's code into a separate js file. The comments don't reach the customer since minification removes them.